docs/src/index.md

```diff
@@ -48,7 +48,7 @@ codes such as [Abinit](https://www.abinit.org/),
       DFTK-based calculator implementation for ASE.
     - Read data in [ETSF Nanoquanta](https://doi.org/10.1016/j.commatsci.2008.02.023) format.
 
-Missing a feature? Look for an open issue or [create a new one](https://github.com/JuliaMolSim/DFTK.jl/issues).
+Missing a feature? Look for an open issue or [create a new one](https://github.com/JuliaMolSim/DFTK.jl/issues). Want to contribute? See the information [here](https://github.com/JuliaMolSim/DFTK.jl#contributing).
```

Comment:
Here is not a good name for a link (too short and meaningless).

Reply:
Oh come on it's a link, this is as clear as can be. Feel free to change it if you don't like it.

Reply:
ok will do